examples/sol-anchor-contract/Anchor.toml

```diff
@@ -0,0 +1,10 @@
+[programs.devnet]
+example_sol_anchor_contract = "4ACQwuQt2ChnPRUV8KwuhVqJvEdusWd3i8JAngHDkp91"
+
+[provider]
+cluster = "devnet"
+wallet = "/home/yunhao/pyth/wallet/pair.json"
```

Comment:
When I call `anchor deploy` it will try to fetch the key from this path.
It doesn't exist in my computer.
I recommend replacing by `~/.config/solana/id.json` (that's a typical path) or saying in the readme that you first need to change this line to point to right path.